src/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+}
+    
+.stars .stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+  
+.stars--4 .stars__star:nth-of-type(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+      
+.stars--4 .stars__star:not(:nth-of-type(-n+4)) {
+  background-image: url(./images/star.svg);
+}

```

Comment:
remain selectors for all variants - what if we will change modifier via js and styles are absent?

Reply:
I hope, I understood the comments correctly